### PR TITLE
Fix CSS in app-navigation

### PR DESCRIPTION
**Pull request by jancborchardt**
This is pretty much done now! Needs some testing with subfolders @nextcloud/mail 

TODO where I need JS help:

- [ ] Don’t show the counter when there are 0 unread mails
- [x] Fix popover position for account settings
- [x] Clicking »Show all folders« and collapse should not reload the main content

screenshot from 2017-11-02 17-26-18

### CODE REVIEW

**Comment by ChristophWurst** [comment]
> Clicking »Show all folders« and collapse should not reload the main content

Wild guess: did you somehow break the click handler binding and the app tries to navigate to `#`?

**Comment by ChristophWurst** [comment]
> Don’t show the counter when there are 0 unread mails

Easy, I can do that. Doesn't have to be in this PR as that's independent.

**Comment by ChristophWurst** [comment]
> Fix popover position for account settings

Why do we need js for this?

File: js/templates/account.html [hunk 1]
```diff
@@ -36,6 +36,8 @@
 <div class="folders"></div>
 {{#unless isUnifiedInbox}}
 {{#if hasFolders}}
-<li class="account-toggle-collapse">{{toggleCollapseMessage}}</li>
+<li class="account-toggle-collapse">
+	<a href="">{{toggleCollapseMessage}}</a>
```
Comment (ChristophWurst) [inline js/templates/account.html]: here you go. The newly added `a` element causes an unexpected navigation. The click handler should call `event.preventDefault()`.

Comment (skjnldsv) [reply js/templates/account.html]: preventDefault isn't working here, wefound something else. ;)

**Comment by jancborchardt** [comment]
Pushed a fix for the other two issues, the position was indeed CSS only. :)

**Comment by ChristophWurst** [comment]
❤️

**Pull request merged**